src/state/notifications/selectors.ts

```diff
@@ -15,6 +18,13 @@ export const sessionToken = createSelector(local, state => state.sessionToken);
 export const notificationError = createSelector(local, state => state.error);
 export const storedEmail = createSelector(local, state => state.email);
 export const storedPin = createSelector(local, state => state.pin);
+export const error = createSelector(local, state => state.error);
+export const enhancedError = createSelector(error, err => {
```

Comment:
Very strange name for error. ReadableError? IDK, minor. There is `notificationError` above, no need to duplicate.

Reply:
I like that, I'll rename